From f612087da830541b4a1398d18742988afbe85a8a Mon Sep 17 00:00:00 2001
From: Nikhil Badola <B46172@freescale.com>
Date: Thu, 28 Nov 2013 20:00:09 +0530
Subject: [PATCH 385/429] powerpc/t4240: Change t4240 USB controller version

Change USB controller version to 2.5 in compatible string for T4240 rev2.0

Signed-off-by: Nikhil Badola <B46172@freescale.com>
Change-Id: Ic6327d4b144549d4ba28bbcf4a99e7161df3587e
Reviewed-on: http://git.am.freescale.net:8181/6956
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Ramneek Mehresh <ramneek.mehresh@freescale.com>
Reviewed-by: Suresh Gupta <suresh.gupta@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
[origin patch is from QorIQ-SDK-V1.5-20131219-yocto]
Signed-off-by: Yang Wei <Wei.Yang@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t4240si-post.dtsi |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
index 965991e..9bd26e5 100644
--- a/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t4240si-post.dtsi
@@ -1236,13 +1236,13 @@
 
 /include/ "qoriq-usb2-mph-0.dtsi"
 		usb0: usb@210000 {
-			compatible = "fsl-usb2-mph-v2.4", "fsl-usb2-mph";
+			compatible = "fsl-usb2-mph-v2.5", "fsl-usb2-mph";
 			phy_type = "utmi";
 			port0;
 		};
 /include/ "qoriq-usb2-dr-0.dtsi"
 		usb1: usb@211000 {
-			compatible = "fsl-usb2-dr-v2.4", "fsl-usb2-dr";
+			compatible = "fsl-usb2-dr-v2.5", "fsl-usb2-dr";
 			dr_mode = "host";
 			phy_type = "utmi";
 		};
-- 
1.7.5.4

